docs/digger/index.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: index
+title: Digger
+section: digger
+---
+
+<h2 class="section-header" id="digger"><img src="/img/aerogeardigger_icon_32px_cropped.png"> AeroGear<strong>Digger</strong> Guides</h2>
+___
+<p>
+<a href="https://github.com/aerogear/digger-installer" class="btn btn-primary-inverse"><i class="fa fa-cloud-download"></i> AeroGear Digger Installer</a>
```

Comment:
The repo name has been changed to `https://github.com/aerogear/aerogear-digger-installer` and could we also make this open a new tab